data/compliance_profiles/OracleLinux/6/nist_800_53_rev4.json

```diff
@@ -0,0 +1,3024 @@
+{
```

Comment:
These files should be symlinks if they are identical

Reply:
Symlinks get rejected by `puppet module build` and the Forge

Reply:
lovely! quality work. have to push up v2 soon and write a splitting script so the duplication and syncing problems can stop.